### PR TITLE
mutate: link to testcase

### PR DESCRIPTION
**Pull request by OliverGreen27**
Added link to testcase pages in filereport

### CODE REVIEW

File: plugin/mutate/data/source.js [hunk 1]
```diff
@@ -239,14 +239,16 @@ function set_info_line(mutid) {
...
+            row.innerHTML = make_td(test_case + ": " + g_testcases_kills[test_case], null, '../test_cases/' + test_case +'.html');
         }
     }
 }
-function make_td(html, id) {
-    if(!id)
-        return "<td>"+html+"</td>";
-    return "<td id='"+id+"'>"+html+"</td>";
+function make_td(html, id, link) {
+    if(id && !link)
```
Comment (joakim-brannstrom) [inline plugin/mutate/data/source.js:247]: space after if

File: plugin/mutate/data/source.js [hunk 2]
```diff
@@ -239,14 +239,16 @@ function set_info_line(mutid) {
         }
         var test_case = tcs[i];
         if (test_case) {
-            row.innerHTML = make_td(test_case + ": " + g_testcases_kills[test_case]);
+            row.innerHTML = make_td(test_case + ": " + g_testcases_kills[test_case], null, '../test_cases/' + test_case +'.html');
```
Comment (joakim-brannstrom) [inline plugin/mutate/data/source.js:242]: space after +

File: plugin/mutate/data/source.js [hunk 3]
```diff
@@ -239,14 +239,16 @@ function set_info_line(mutid) {
...
     }
 }
-function make_td(html, id) {
-    if(!id)
-        return "<td>"+html+"</td>";
-    return "<td id='"+id+"'>"+html+"</td>";
+function make_td(html, id, link) {
+    if(id && !link)
+        return "<td id='"+id+"'>"+html+"</td>";
+    if(link)
```
Comment (joakim-brannstrom) [inline plugin/mutate/data/source.js:249]: space

**Comment by OliverGreen27** [comment]
When PR "mutate: testcase name filter" is merged I will update this branch and change small bits. 

This PR is not ready for merge yet

**Comment by joakim-brannstrom** [comment]
finalized in #1643 . thanks for the work :)

**Pull request closed without merging**